packages/components/src/components/tooltip/tooltip.css

```diff
@@ -67,6 +67,7 @@
 
 [part='arrow'] {
   position: absolute;
+  z-index: -1; /* always below the text */
```

Comment:
so this might cause the arrow to disappear under another element, right? The tooltip itself uses `--scl-z-index-70` - shall we just make it `70-1` for the arrow?

Reply:
nice catch, I'm going to test this… I think that -1 might be relative to 70 (within the "scope" of the tooltip element) 🤔

Reply:
works as expected, can stay `-1` 👍 — thanks again for asking